settlement/tests/treasury.rs

```diff
@@ -102,12 +116,12 @@ impl MythereumTreasuryContract {
                 amount,
                 receiver_address,
             }) => {
-                if token_address != "tether-address" {
+                if token_address != string_to_hex("tether-address") {
```

Comment:
If we use string_to_hex, the encoded data is the same as the previous one except for type. Is it intended?